app/config/app-template.js

```diff
@@ -11,6 +11,9 @@ export const SEARCH_SLUG = 'search';
 
 // Starward
 export const baseURL = `http://${HOST}:${PORT}`;
+export const WP_URL = 'http://localhost/starward_wp';
+export const WP_RESOURCE_URL = WP_URL; // change to '' and images will attempt to resolve to the app host, then if you have nginx setup you can proxy the WP through the APP hosting's address
```

Comment:
this new config value is required so that we can per-environment change where we fetch images. This allows us to proxy images through someawesomesite.com instead of wp.someawesomesite.com